curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148da157cc0bd0d06df5c0a.md

```diff
@@ -31,9 +31,17 @@ assert.equal(display, 'inline-block');
 You should give the `label` element a `text-align` of `right`.
 
 ```js
-const gs = (s) => new __helpers.CSSHelp(document).getStyle(s)?.textAlign;
-const textAlign = gs('.info > label') ?? gs('.info label');
-assert.equal(textAlign, 'right');
+const el = document.querySelectorAll('.info label');
+let result = 'right';
+for(let i=0; i<el.length; i++){
+	const style = getComputedStyle(el[i])
+    const value = style.getPropertyValue('text-align')
+    if(value !== 'right'){
+      result = value;
+      break;      
+    }
+}
+assert.equal(result, 'right');
```

Comment:
```suggestion
const v = (el) => getComputedStyle(el).getPropertyValue('text-align');
document.querySelectorAll('.info label').forEach(el => {
  assert.equal(v(el), 'right');
});
```

Reply:
I thought `forEach` don't hold reference to DOM node. I should have checked it 😥